components/handlers/fixed_module_mountings.py

```diff
@@ -97,5 +97,6 @@ def POST(self):
             This method is invoked when users click on the button
             to navigate to the fixed module mountings, that is
             present in other valid pages.
+            used by module mountingtentative
```

Comment:
Can consider changing the navigation buttons of fixed/tentative to use javascript onclick event instead of post method. Then can safely remove these post methods